read-cache.c

```diff
@@ -2229,7 +2229,7 @@ static int should_delete_shared_index(const char *shared_index_path)
 	if (!expiration)
 		return 0;
 	if (stat(shared_index_path, &st))
-		return error_errno(_("could not stat '%s"), shared_index_path);
+		return error_errno(_("could not stat '%s'"), shared_index_path);
```

Comment:
Good catch, but @nafmo should send this patch (on `read-cache.c`) to Git mailing list for review, and it will be better to have prefix "i18n: " in the title. 

We should know our l10n team should not touch any other files outside po/ directory.